jekyll/_cci2/vs-code-extension-quickstart.adoc

```diff
@@ -0,0 +1,117 @@
+---
+contentTags: 
+  platform:
+  - Cloud
+---
+= Get started with the VS Code extension
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A quickstart tutorial for the CircleCI VS Code extension.
+:icons: font
+:toc: macro
+:toc-title:
+
+This tutorial is a quickstart guide to setting up and using the official CircleCI VS Code extension. You can read more about the extension's capabilities in the xref:vs-code-extension-overview#[overview]. 
+
+[#prerequisits]
+== Prerequisites
+
+Before starting this tutorial, make sure you have:
+
+* Installed the Visual Studio Code editor
```

Comment:
My first thought reading this was "What is the Visual Studio Code editor", because nobody calls it this. Made me think it's a separate thing. Can we call it VS Code?

We could even replace the whole bullet point with 
* VS Code version 1.67 or above
(instead of splitting into bullet + info box)

Reply:
I agree with ^ just specify the version as part of the requirement